### PR TITLE
Improved Debug Output

### PR DESCRIPTION
**Pull request by abrown-sg**
* Passing --debug to molecule now displays config, environment (separated into ansible and other groups) and ansible-playbook command.

* Refactored converge code slightly to remove duplication. Multiple return values weren't being used and were unnecessary.

### CODE REVIEW

File: molecule/commands.py [hunk 1]
```diff
@@ -154,15 +155,16 @@ def converge(self, idempotent=False, create_instances=True, create_inventory=Tru
...
-                output = sh.ansible_playbook(playbook, *args, **kwargs)
-                return output
-            except sh.ErrorReturnCode as e:
-                print('ERROR: {}'.format(e))
-                sys.exit(e.exit_code)
+
         try:
             output = sh.ansible_playbook(playbook, *args, **kwargs)
-            return output.exit_code
+            if self.molecule._args['--debug']:
```
Comment (retr0h) [inline molecule/commands.py]: Can we report the output prior to execution.  Would be nice to see what we will be doing vs waiting to converge to display this.

File: molecule/utilities.py [hunk 2]
```diff
@@ -25,6 +25,8 @@
 import sys
 
 from colorama import Fore
+from colorama import Back
```
Comment (rgreinho) [inline molecule/utilities.py]: Put ``Back`` before  ``Fore``

**Comment by rgreinho** [comment]
Question: Does ``init(autoreset=True)`` work in our use cases? If no, disregard. If yes, why not using it?

File: molecule/commands.py (hunk 1 above)
Comment (abrown-sg) [inline molecule/commands.py]: Done!

File: molecule/utilities.py (hunk 2 above)
Comment (abrown-sg) [inline molecule/utilities.py]: Fixed!

**Comment by abrown-sg** [comment]
Good find with colorama! I'll take a look at doing that in a future update.

**Comment by retr0h** [comment]
:+1:

**Pull request merged**